pmm-app/src/pmm-update-panel/components/CenteredButton/CenteredButton.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Button, ButtonProps } from '@grafana/ui';
+
+import * as styles from './CenteredButton.styles';
+
+export const CenteredButton = ({ children, className, ...props }: ButtonProps) => (
+  <Button className={`${className} ${styles.centeredButton}`} {...props}>
```

Comment:
What if I wand to pass className as undefined? Pls check out `import { cx } from 'emotion';`. It's properly handle undefined )

Reply:
Neat! Thanks, done